build.cmd

```diff
@@ -0,0 +1,14 @@
+@echo off
```

Comment:
Paket doesn't care. This is how it is mostly tends to be done in F# OSS projects. Mainly, I think, because there is no configuration. PS doesn't work unless you've fiddled with security settings. The code is now in cross-plat .fsx so PS doesn't really give us anything, but complicates things for users who aren't used to PS.